app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> bool:
+    c_args = command.split()
+    if command == "" or c_args[0] != "mv" or c_args[1] == c_args[2]:
+        return False
+    file1, file2 = c_args[1], c_args[2]
```

Comment:
You can do this in `split()` function like 
`cmd, file1, file2 = command.split()`